src/utils/tests/parser.js

```diff
@@ -57,8 +58,15 @@ const classTest = formatCode(`
   }
 `);
 
+const varTest = formatCode(`
+  var foo = 1;
+  let bar = 2;
+  const baz = 3;
+  const a = 4, b = 5;
+`);
```

Comment:
perhaps we should find params and properties as well:

`function(bar) {}`
`x = {a: 2}`